.buildkite/gpu_pipeline.yml

```diff
@@ -39,18 +39,21 @@ steps:
 
       echo "--- :python: Test"
       BAZEL_BUILD_FLAGS=()
-      GCC_VERSION=$(gcc --version | head -1 | awk '{ print $3 }' | cut -d. -f1)
-      if [[ "${GCC_VERSION}" -le 12 ]]; then
+      GCC_VERSION=\$(gcc --version | head -1 | awk '{ print \$3 }' | cut -d. -f1)
+      if [[ "\${GCC_VERSION}" -le 12 ]]; then
         # Someone wants to compile some code which requires flags not understood by GCC 12.
         BAZEL_BUILD_FLAGS+=(--define=xnn_enable_avxvnniint8=false)
       fi
-      if [[ "${GCC_VERSION}" -le 11 ]]; then
+      if [[ "\${GCC_VERSION}" -le 11 ]]; then
         # Someone wants to compile some code which requires flags not understood by GCC 11.
+        BAZEL_BUILD_FLAGS+=(--define=xnn_enable_avxvnni=false)
```

Comment:
As mentioned at https://github.com/EnzymeAD/Enzyme-JAX/pull/295#issuecomment-2620321642, this is only for GCC < 10, for 11 you don't need it.  I presume you're using GCC 10 or something.

Reply:
ah right, it seems the worker has gcc v9